x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/allowed_values/helpers.test.tsx

```diff
@@ -203,14 +148,5 @@ describe('helpers', () => {
         },
       ]);
     });
-
-    test('it returns an empty array when `ecsMetadata` is null', () => {
-      expect(
-        getUnallowedValueRequestItems({
-          ecsMetadata: null, // <--
-          indexName: 'auditbeat-*',
-        })
-      ).toEqual([]);
-    });
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631493542